SPEC.md

```diff
@@ -82,7 +82,7 @@ In this case, the ACI is both compressed and encrypted.
 
 ```
 tar cvf reduce-worker.tar manifest rootfs
-gpg --output reduce-worker.sig --detach-sig reduce-worker.tar
+gpg --output reduce-worker.aci.asc --detach-sig reduce-worker.tar
```

Comment:
Change to: ```gpg --armor --output ```... so that this command always outputs an ascii-armored signature (otherwise this might be relying on specific version/configuration of gpg). Call out that it should (must?) be ascii-armored on line 73?

> - Image archives SHOULD be signed using PGP, in detached signature mode.